docs/UPGRADING.rst

```diff
@@ -0,0 +1,17 @@
+Upgrading HaaS
+==============
+
+This file describes the procedure for upgrading HaaS to a new version.
+
+1. Read the release notes, as they may contain version-specific information.
+2. Stop the HaaS services (webserver and HaaS network daemon).
+3. Download and install the new version of HaaS::
+
+     $ git clone https://github.com/cci-moc/haas
+     $ python.setup.py install
```

Comment:
there is a typo in the command.

Also does running `python setup.py install` overwrites the executable with newer version ?
I am guessing that there should be some command for uninstalling old code before running this command.